src/Microsoft.VisualStudio.Composition/RuntimeComposition.cs

```diff
@@ -227,15 +226,15 @@ private static RuntimeExport CreateRuntimeExport(ExportDefinitionBinding exportD
             Requires.NotNull(exportDefinitionBinding, nameof(exportDefinitionBinding));
             Requires.NotNull(resolver, nameof(resolver));
 
-            var partDefinitionType = exportDefinitionBinding.PartDefinition.TypeRef.Resolve();
+            var partDefinitionTypeRef = exportDefinitionBinding.PartDefinition.TypeRef;
             return CreateRuntimeExport(
                 exportDefinitionBinding.ExportDefinition,
-                partDefinitionType,
+                partDefinitionTypeRef,
                 exportDefinitionBinding.ExportingMemberRef,
                 resolver);
         }
 
-        [DebuggerDisplay("{" + nameof(RuntimePart.TypeRef) + "." + nameof(Reflection.TypeRef.ResolvedType) + ".FullName,nq}")]
+        [DebuggerDisplay("{" + nameof(RuntimePart.TypeRef) + ".FullName,nq}")]
```

Comment:
tip: use `nameof(TypeRef.FullName)` here because it's referencing a symbol we define ourselves and might change. We didn't do this before because it was *Type*.FullName, which will never change. But *TypeRef*.FullName might be removed/renamed in the future.

Reply:
The way you've fixed this, you've removed "TypeRef." from the string. Only the last member (FullName) contributes to the string.

---
In reply to: [262675122](https://github.com/Microsoft/vs-mef/pull/111#discussion_r262675122) [](ancestors = 262675122)